pyproject.toml

```diff
@@ -27,6 +27,7 @@ parameterized = "^0.9.0"
 yara-python = "^4.5.0"
 idstools = "^0.6.5"
 aenum = "^3.1.15"
+boto3 = "^1.35.22"
```

Comment:
I made `boto3` an optional dependency, I think that this might work better as an `extra` rather than an `option-group` seems as though [extras](https://python-poetry.org/docs/pyproject/#extras) are meant for one off dependencies and features this.

Instructions for installations for this would now be:
```
poetry install --extras s3
```

Added in [e218106](https://github.com/yeti-platform/yeti/pull/1137/commits/e2181069ce09378055c1599d73a9d2558b6e32b1)